hooks/command

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
```

Comment:
Do you know if this is still required on the newer Buildkite agents?

Reply:
I'll get this in for now but will make a note to check this 👍